examples/labs/orientation/orientation.h

```diff
@@ -42,70 +42,70 @@
 #include "psmove_tracker.h"
 #include "psmove_tracker_opencv.h"
 
-extern "C" {
-    void cvShowImage(const char *, void*);
-};
-
 class Orientation : public QThread
 {
     Q_OBJECT
+signals:
+    void orientation(qreal a, qreal b, qreal c, qreal d,
+                     qreal scale, qreal x, qreal y);
+
+public:
+    Orientation() : QThread() {}
+
+    void run() override
+    {
+        PSMove *move = psmove_connect();
+        int quit = 0;
+        float q0, q1, q2, q3;
+
+        if (move == NULL) {
+            fprintf(stderr, "Could not connect to controller.\n");
+            QMetaObject::invokeMethod(qApp, "quit", Qt::QueuedConnection);
```

Comment:
Can this also use the function pointer style syntax?

```c++
template <typename Functor, typename FunctorReturnType>
bool QMetaObject::invokeMethod(QObject *context, Functor function,
    Qt::ConnectionType type = Qt::AutoConnection,
    FunctorReturnType *ret = nullptr);
```


> Invokes the function in the event loop of context. function can be a functor or a pointer to a member function. Returns true if the function could be invoked. Returns false if there is no such function or the parameters did not match. The return value of the function call is placed in ret.

https://doc.qt.io/archives/qt-5.15/qmetaobject.html#invokeMethod-4